misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -319,12 +329,13 @@ private CoverageBuilder analyzeStructure() throws IOException {
     }
 
     private void createReport(final IBundleCoverage bundleCoverage, Map<String, ModuleCoverage> moduleCoverageMap) {
-        createReport(bundleCoverage, moduleCoverageMap, null);
+        createReport(bundleCoverage, moduleCoverageMap, null, null);
     }
 
     private void createReport(final IBundleCoverage bundleCoverage, Map<String, ModuleCoverage> moduleCoverageMap,
-                              List<DocumentId> exclusionList) {
+                              List<DocumentId> exclusionList, List<String> exclusionClassList) {
         boolean containsSourceFiles = true;
+        Set<String> externalExclusionList = new HashSet<>(exclusionClassList);
```

Comment:
Can't we pass a Set from the beginning without having to convert a list to a set?